test_unstructured_ingest/expected-structured-output/Sharepoint-with-permissions/None

```diff
@@ -1,25 +1,25 @@
 [
   {
-    "element_id": "32bc8af17151389d3e80f65036f8e65b",
+    "type": "Table",
```

Comment:
This is a strange document name None...
I don't see it in the current expected-output in main

Reply:
Oh it renamed it...